### PR TITLE
Design Bugs

### PR DESCRIPTION
**Pull request by tuantruong625**
Reference #162.

Screen Shot 2019-05-09 at 3 26 38 PM

To fix button sizes in header, had to use custom css.

Fixes some of the bigger rendering issues: General layout/size/padding issuyes, rich text editor rendering, etc. Rest is still WIP.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -48,9 +47,9 @@
                                           v-on:css-errors="cssErrors = $event"/>
                   </div>
               </div>
-            </div>
+            </b-col>
 
-            <div class="data-container col-4 border overflow-auto pr-0 pl-0">
+            <b-col cols="4" class="data-container border overflow-auto pr-0 pl-0 rounded">
```
Comment (velkymx) [inline src/App.vue]: This should only need the card wrapper! Why are we adding all of these classes???

Comment (velkymx) [reply src/App.vue]: https://getbootstrap.com/docs/4.3/components/card/

**Comment by velkymx** [comment]
Here's the HTML from the New Design ticket.

**Comment by velkymx** [comment]
Getting an error when I do npm run serve

```
Module Warning (from ./node_modules/eslint-loader/index.js):
error: 'v-model' directives aren't supported on <button> elements (vue/valid-v-model) at src/components/renderer/form-button.vue:3:64:
  1 | <template>
  2 |     <div class="form-group">
> 3 |         <button @click="click" :class="classList" :name="name" v-model="fieldValue">{{ label }}</button>
    |                                                                ^
  4 |     </div>
  5 | </template>
  6 | 
```

**Comment by ssshake** [comment]
button @click="click" :class="classList" :name="name" v-model="fieldValue"

This was a pre-existing issue and does not break the app, just throws a warning. Prior discussions led to leaving this as is in this PR as it's out of scope and then fixing it outside of this PR.

I've split this out into a separate issue: https://github.com/ProcessMaker/spark-screen-builder/issues/175

**Comment by caleeli** [comment]
Should we have to do any additional change in Spark, I see that this style is still wrong:

The border of the page selector is still square

image

Screenshot 2019-05-07 10 06 50

**Comment by velkymx** [comment]
@caleeli don't worry about that style on the dropdown for now.

**Pull request merged**